README.md

```diff
@@ -1,44 +1,34 @@
-# Executor Router
+# Executor Queue
 [![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][status-image]][status-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
 
-> A generic executor plugin that routes builds to a specified executor
+> A generic executor plugin that routes builds through a queue
```

Comment:
This isn't *really* generic. It is very specifically routing builds through a redis queue.